app/javascript/components/textual_summary/table_list_view.jsx

```diff
@@ -30,9 +30,8 @@ export default function TableListView(props) {
 
   /** Function to generate rows for structured list. */
   const miqListRows = (list) => {
-    const data = [];
-    list.map((item) => data.push({ ...item, label: item.name }));
-    return data;
+    const headerKeys = props.headers.map((item) => (item.toLowerCase().replaceAll(/\s/g, '_')));
+    return list.map((item) => (headerKeys.map((header) => item[header])));
```

Comment:
@jeffibm This is currently returning a 5x5 array of `undefined` for me.
![image](https://user-images.githubusercontent.com/64800041/236552556-be6e246d-deae-4e79-aaff-c110e81d4c9f.png)

Reply:
> @jeffibm This is currently returning a 5x5 array of `undefined` for me.
> ![image](https://user-images.githubusercontent.com/64800041/236552556-be6e246d-deae-4e79-aaff-c110e81d4c9f.png)

Hey @DavidResende0,
I am able to see the all the data as expected. this is how I get in MiqStrucuredList component

<img width="889" alt="image" src="https://user-images.githubusercontent.com/87487049/236673542-e86c3b8b-f524-4e89-8a81-bff77055b60f.png">

you must be getting something like below
<img width="643" alt="image" src="https://user-images.githubusercontent.com/87487049/236674016-adf6ffc5-dcff-4377-b8d8-cc580d5b1105.png">

could you please check what do you get for **`rows`** in here
`app/helpers/ops_helper/textual_summary.rb`
 `def get_tenant_quota_allocations`

Reply:
Hey @DavidResende0 ,
the previous fix will not work with translations and I had made a fix for it.
could you please pull and test again when you have time?

Reply:
Looks like the fix worked